Source/Core/Common/GL/GLInterface/AGL.mm

```diff
@@ -49,15 +33,34 @@
     return false;
   }
 
-  if (cocoaWin == nil)
+  if (window_handle)
```

Comment:
I sort of like the visual distinction the indent provides, it's easy to see which part is only used with a window. What do you think?